OKD.md

```diff
@@ -0,0 +1,85 @@
+# [kubevirtci](README.md): Getting Started with multi-node OKD Provider
```

Comment:
This is all the same for OCP maybe we can say so.

Reply:
IDK, anyone using OCP needs access to the quay registry? So should we?